### PR TITLE
Add table gcp_compute_disk_metric_read_ops_hourly #223

### PR DESCRIPTION
**Pull request by rajeshbal65**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
### Basic info

```sql
select
  name,
  minimum,
  maximum,
  average,
  sample_count
from
  gcp_compute_disk_metric_read_ops_hourly
order by
  name;

+------------+---------+---------+----------------------+--------------+
| name       | minimum | maximum | average              | sample_count |
+------------+---------+---------+----------------------+--------------+
| disk-1     | 0       | 0       | 0                    | 61           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 57      | 3.057142857142857    | 35           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 74      | 2.4318181818181817   | 44           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 60           |
| disk-1     | 0       | 0       | 0                    | 61           |
| instance-1 | 0       | 0       | 0                    | 61           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 23      | 0.7                  | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 4006    | 81.40677966101696    | 59           |
| instance-1 | 0       | 0       | 0                    | 61           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 2468    | 136.5142857142857    | 35           |
| instance-1 | 0       | 2       | 0.06666666666666667  | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 558     | 17.866666666666667   | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 1       | 0.016666666666666666 | 60           |
| instance-1 | 0       | 4698    | 188.23333333333332   | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 61           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 3228    | 108.45454545454545   | 44           |
| instance-1 | 0       | 3620    | 188.3                | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 456     | 10.8                 | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 1       | 0.03333333333333333  | 60           |
| instance-1 | 0       | 304     | 7.883333333333334    | 60           |
| instance-1 | 0       | 1       | 0.016666666666666666 | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
| instance-1 | 0       | 0       | 0                    | 60           |
+------------+---------+---------+----------------------+--------------+
```

### Intervals averaging over 100 connections

```sql
select
  name,
  round(minimum::numeric,2) as min_read_ops,
  round(maximum::numeric,2) as max_read_ops,
  round(average::numeric,2) as avg_read_ops,
  sample_count
from
  gcp_compute_disk_metric_read_ops_hourly
where average > 100
order by
  name;

+------------+--------------+--------------+--------------+--------------+
| name       | min_read_ops | max_read_ops | avg_read_ops | sample_count |
+------------+--------------+--------------+--------------+--------------+
| instance-1 | 0.00         | 2468.00      | 136.51       | 35           |
| instance-1 | 0.00         | 4698.00      | 188.23       | 60           |
| instance-1 | 0.00         | 3228.00      | 108.45       | 44           |
| instance-1 | 0.00         | 3620.00      | 188.30       | 60           |
+------------+--------------+--------------+--------------+--------------+
```

### Intervals averaging fewer than 10 connections

```sql
select
  name,
  round(minimum::numeric,2) as min_read_ops,
  round(maximum::numeric,2) as max_read_ops,
  round(average::numeric,2) as avg_read_ops,
  sample_count
from
  gcp_compute_disk_metric_read_ops_hourly
where average < 10
order by
  name;

+------------+--------------+--------------+--------------+--------------+
| name       | min_read_ops | max_read_ops | avg_read_ops | sample_count |
+------------+--------------+--------------+--------------+--------------+
| disk-1     | 0.00         | 0.00         | 0.00         | 61           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 57.00        | 3.06         | 35           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 74.00        | 2.43         | 44           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 60           |
| disk-1     | 0.00         | 0.00         | 0.00         | 61           |
| instance-1 | 0.00         | 0.00         | 0.00         | 61           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 2.00         | 0.07         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 1.00         | 0.02         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 61           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 1.00         | 0.03         | 60           |
| instance-1 | 0.00         | 304.00       | 7.88         | 60           |
| instance-1 | 0.00         | 1.00         | 0.02         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 61           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 0.00         | 0.00         | 60           |
| instance-1 | 0.00         | 23.00        | 0.70         | 60           |
+------------+--------------+--------------+--------------+--------------+
```
</details>

### CODE REVIEW

File: gcp/table_gcp_disk_metric_read_ops_hourly.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	"google.golang.org/api/compute/v1"
+)
+
+//// TABLE DEFINITION
+
+func tableGcpDiskMetricReadOpsHourly(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_disk_metric_read_ops_hourly",
+		Description: "GCP Compute Disk Daily connections",
```
Comment (Subhajit97) [inline gcp/table_gcp_disk_metric_read_ops_hourly.go:17]:
```suggestion
		Description: "GCP Compute Disk Metrics -Read Ops (Hourly)",
```

File: gcp/table_gcp_disk_metric_read_ops_hourly.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+				Transform:   transform.FromField("DimensionValue"),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listDiskMetricReadOpsHourly(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	instanceInfo := h.Item.(*compute.Instance)
```
Comment (Subhajit97) [inline gcp/table_gcp_disk_metric_read_ops_hourly.go:36]: Is it correct?
Why are we fetching `Instance` details, since the table stores `disk` metrics?

**Review by Subhajit97 — CHANGES_REQUESTED** [review]
@rajeshbal65 Please verify the table and test properly

**Review by Subhajit97 — APPROVED** [review]

**Review by rajmohanty17 — APPROVED** [review]
LGTM

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**